services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -112,8 +112,14 @@ class ProtectedPathSpec extends BaseSpecification {
                     !Cookie.isDefaultRule(it.cookie)
                 }
 
-                assert flowHelper.findForwardPathRules(rules, sw).size() == 0
-                assert flowHelper.findReversePathRules(rules, sw).size() == 0
+                assert !rules.findAll {
+                    it.match.inPort == sw.inputPort.toString() &&
+                            it.instructions.applyActions.flowOutput == sw.outputPort.toString()
```

Comment:
look for cookie instead? This is not critical, but port match is not enough to ensure it is the right rule.